helm/kadalu/templates/resources.yaml

```diff
@@ -23,6 +23,13 @@ spec:
             spec:
               type: object
               properties:
+                disperse:
```

Comment:
Not sure why CI failed to find the difference between manifests and helm templates, maybe CRD isn't being generated as apart of template in CI. Nevertheless, can you please also run `make gen-manifest`?

Reply:
Done.